compiler/luci-micro/helpers/GenerateKernelsListHelper.cpp

```diff
@@ -157,7 +157,7 @@ std::vector<char> loadFile(const std::string &path)
   if (!file.good())
   {
     std::string errmsg = "Failed to open file: " + path;
-    throw std::runtime_error(errmsg.c_str());
+    assert(false && errmsg.c_str());
```

Comment:
Could you rewrite these lines, like it is done below?
`assert` will not print error message as intended.